web/core/components/pages/editor/editor-body.tsx

```diff
@@ -176,7 +176,7 @@ export const PageEditorBody: React.FC<Props> = observer((props) => {
               }}
               handleEditorReady={handleEditorReady}
               ref={editorRef}
-              containerClassName="p-0 pb-64"
+              containerClassName="h-full p-0 pb-64"
```

Comment:
It should be min-h-screen?

Reply:
Not needed as it would add an unnecessary scroll to the container since the editor doesn't always occupy the complete screen height.